platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java

```diff
@@ -0,0 +1,885 @@
+    manageLogInOut.signOut();
+  }
+
+  /**
+   * <li>Case ID:121915.</li>
+   * <li>Test Case Name: Mention a user in activity composer.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> Step Number: 1 Step Name: Step 1: Mentions on User
+   * Activity Stream Step Description: - Login Intranet home - Hover the mouse
+   * over the name of user and select My Activity streams - Write something
+   * -Entering @ followed by the user name - Select name is wrapped in the input -
+   * Click Share button Input Data: Expected Outcome: - The suggestion list is
+   * hidden - In the activity stream, mentions are displayed as a link on
+   * "Firstname Lastname" to the user's activities page
+   */
+  @Test
+  public void test04_MentionAUserInActivityComposer() {
+    info("Test 4: Mention a user in activity composer");
+    String text = "text" + getRandomNumber();
+    homePagePlatform.goToHomePage();
+    activityStream.mentionUserActivity(DATA_USER2, text);
+    $(byText(text)).parent().parent().find(byClassName(ELEMENT_DATE_ACTIVITY)).click();
+    $(byText(text)).parent().parent().find(ELEMENT_ICON_DELETE_ACTIVITY).click();
+    $(ELEMENT_DELETE_POPUP_OK).click();
+    // verify that the activity doesn't exist
+    $(byText(text)).shouldNot(Condition.exist);
+  }
+
+  /**
+   * <li>Case ID:121923.</li>
+   * <li>Test Case Name: Share your status.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> Step Number: 1 Step Name: Share your status Step
+   * Description: - Log in - Go to Home page - Input into activity input field and
+   * click share Input Data: Expected Outcome: - The message will show in Activity
+   * Stream
+   */
+  @Test
+  public void test05_ShareYourStatus() {
+    info("Test 5: Share your status");
+
+    String name = "name" + getRandomNumber();
+
+    homePagePlatform.goToHomePage();
+    activityStream.addActivity(name, "");
+    $(byText(name)).should(Condition.exist);
+    $(byText(name)).parent().parent().find(byClassName(ELEMENT_DATE_ACTIVITY)).click();
+    $(byText(name)).parent().parent().find(ELEMENT_ICON_DELETE_ACTIVITY).click();
+    $(ELEMENT_DELETE_POPUP_OK).click();
+    // verify that the activity doesn't exist
+    $(byText(name)).shouldNot(Condition.exist);
+  }
+
+  /**
+   * <li>Case ID:121924.</li>
+   * <li>Test Case Name: Add a document.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> Step Number: 1 Step Name: - Go to Select File Dialog
+   * Step Description: - Log in - Goto Homepage - In Activity Composer click File
+   * (icon) Input Data: Expected Outcome: - Select File Dialog shows up
+   */
+
+  /*
+   * Step number: 2 Step Name: - Select document Step Description: - Browse the
+   * Select File Dialog to choose a document - Click Select Input Data: Expected
+   * Outcome: - A breadcrumb is displaying the current position of the user in the
+   * browsed drive. - the name of the file should be displayed under the activity
+   * input field.
+   */
```

Comment:
Comments should be merged (one comment block per method) for a better readability.

Reply:
done